src/scipp/plotting/resampling_model.py

```diff
@@ -144,8 +144,12 @@ def _rebin(self, var, coords, *, sanitize=False):
             else:
                 plan.insert(0, edge)
         for edge in plan:
+            if edge.unit is None:
+                # We cannot rebin with such a unit.
+                edge = edge.copy()
+                edge.unit = ''
```

Comment:
I don't understand this solution, can you explain what will happen if we don't resample?

Reply:
You mean if we remove the resampling behaviour at a later stage or if we are in a case were no resampling is performed?
For the latter, I didn't think we could not be resampling if we are inside a function called `_rebin`?
But I have to say I have not looked at this code for a while, and I fell now that I understand it less and less.

Reply:
I think the comment in line 148 misled me: I thought you are now skipping rebinning if there is no unit, but what you mean is that you simply set the unit to allow for rebinning?